packages/plans-grid-next/src/components/shared/plan-actions.tsx

```diff
@@ -528,14 +528,14 @@ const PlanFeaturesActionsButton: React.FC< PlanFeaturesActionsButtonProps > = (
 	);
 };
 
-const PlanFeatures2023GridActions = ( props: PlanFeaturesActionsButtonProps ) => {
+const PlanGridActions = ( props: PlanActionsButtonProps ) => {
```

Comment:
This too - probably best for the export to match the name of the file, for consistency. It's `PlanGridActions` then file is `plan-actions`, etc.